### PR TITLE
Fix Metricbeat doc build

### PR DESCRIPTION
**Pull request by ruflin**
* Adjust title levels
* Add zookeeper to doc build
* Add automatic links and references for modules and metricsets

### CODE REVIEW

File: metricbeat/module/zookeeper/mntr/_beat/docs.asciidoc [hunk 1]
```diff
@@ -1,3 +1,3 @@
-==== Zookeeper mntr
+=== Zookeeper mntr MetricSetm
```
Comment (andrewkroh) [inline metricbeat/module/zookeeper/mntr/_beat/docs.asciidoc]: Trailing `m` on this line

File: metricbeat/module/system/cores/_beat/docs.asciidoc [hunk 2]
```diff
@@ -1,3 +1,3 @@
-==== System Cores
+=== System Cores Metricset
```
Comment (andrewkroh) [inline metricbeat/module/system/cores/_beat/docs.asciidoc]: Inconsistent casing of `Metricset` vs `MetricSet`.

File: metricbeat/docs/modules/zookeeper/mntr.asciidoc [hunk 3]
```diff
@@ -3,15 +3,15 @@ This file is generated! See scripts/docs_collector.py
 ////
 
 [[metricbeat-metricset-zookeeper-mntr]]
-==== Zookeeper mntr
+=== Zookeeper mntr MetricSetm
```
Comment (andrewkroh) [inline metricbeat/docs/modules/zookeeper/mntr.asciidoc]: Trailing `m`

File: metricbeat/docs/modules/system/cores.asciidoc [hunk 4]
```diff
@@ -3,15 +3,15 @@ This file is generated! See scripts/docs_collector.py
 ////
 
 [[metricbeat-metricset-system-cores]]
-==== System Cores
+=== System Cores Metricset
```
Comment (andrewkroh) [inline metricbeat/docs/modules/system/cores.asciidoc]: `s/Metricset/MetricSet/`

**Comment by ruflin** [comment]
Issues fished and new version pushed.

**Pull request merged**